eventLab.js

```diff
@@ -0,0 +1,32 @@
+document.addEventListener("DOMContentLoaded", () => {
+
+    let form = document.querySelector("form"); // grabs the form
+    form.addEventListener("submit", (e) => { // event listner on the form's submit button
+        
+        let userInput = document.querySelector("#userInput")
+        let input = userInput.value
+        let ul = document.querySelector("#ul");
+        let li = document.createElement("li");
+        li.innerText = input
+        let error = document.createElement("p");//Creates a paragraph
+
+
+        if(input === ""){  // if the input is empty strings 
```

Comment:
This works but try using `input.trim()` the trim method removes empty spaces in a string and returns the remaining value, then compare that to  `""`